dashboard/app/models/pd/teachercon1819_registration.rb

```diff
@@ -40,7 +40,7 @@ def update_application_status
   def send_teachercon_confirmation_email
     return unless pd_application.workshop && pd_application.workshop.teachercon?
 
-    Pd::Teachercon1819RegistrationMailer.confirmation(self).deliver_now
+    Pd::Teachercon1819RegistrationMailer.send(pd_application.application_type.downcase, self).deliver_now
```

Comment:
we'll want to do something else both here and on the early return for regional partners, since they don't have pd_applications

something like

```ruby
if regional_partner
  Pd::Teachercon1819RegistrationMailer.regional_partner(self).deliver_now
elsif pd_application.workshop && pd_application.workshop.teachercon?
  Pd::Teachercon1819RegistrationMailer.send(pd_application.application_type.downcase, self).deliver_now
end
```